winsup/cygwin/environ.cc

```diff
@@ -953,12 +1012,13 @@ struct spenv
 static NO_COPY spenv spenvs[] =
 {
 #ifdef DEBUGGING
-  {NL ("CYGWIN_DEBUG="), false, true, NULL},
+  {NL ("MSYS_DEBUG="), false, true, NULL},
```

Comment:
This hunk probably wanted to be dropped, as per https://github.com/msys2/msys2-runtime/commit/38020a7e0e6a105b9d4bc25e745286014a36d17d#diff-58d9d402d5db25b56c1a6d36aaa6b8af563f298144a471eaa82b23053bb96580L1015-R1015.

Reply:
Like [this](https://github.com/msys2/msys2-runtime/pull/225/commits/9b4cf2936a50d91759cc60a2ef13e51731a9ca5a).